src/main/kotlin/dev/arbjerg/ukulele/command/QueueCommand.kt

```diff
@@ -12,7 +12,7 @@ import org.springframework.stereotype.Component
 @Component
 class QueueCommand(
         private val players: PlayerRegistry
-) : Command("queue", "q", "list", "l") {
+) : Command("queue", "q") {
```

Comment:
No need to remove alias options, imo.

Reply:
`l` is more important for list than for looping.